docs/quickstart.rst

```diff
@@ -29,11 +30,47 @@ Create *quickstart.py* file and copy and paste the following code.
     gauth = GoogleAuth()
     gauth.LocalWebserverAuth() # Creates local webserver and auto handles authentication.
 
-Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
+Run this code with *python quickstart.py* and you will see a web browser asking you for authentication. 
+Click *Accept* and you are done with authentication. For more details, take a look at documentation: `OAuth made easy`_
 
 .. _`APIs Console`: https://console.developers.google.com/iam-admin/projects
 .. _`OAuth made easy`: ./oauth.html
 
+Authentication via service accounts
```

Comment:
I think it's technically also part of the OAuth 2.0 - https://developers.google.com/identity/protocols/oauth2/service-account